wasm/scripts/compile.sh

```
@@ -1,17 +1,19 @@
-    nix-build -E '(with import <nixpkgs> {}; pkgs.callPackage ./src/plugin_example.nix {})' -o result &&
-    cp ./result/lib/* lib &&
-    # chmod 0600 lib/* &&
-    # nix-build -E '(with import <nixpkgs> {}; pkgs.callPackage ./src/plugin_example_cxx.nix {})' -o result &&
-    # cp ./result/lib/* lib &&
-    printf '%s\n' "wasm binary ready in ./lib"
+    cp ./result/lib/csound.wasm lib &&
+    cp ./result/lib/csound.wasm.z lib &&
+    cp ./result/lib/csound-plugin-sdk.tar.gz lib &&
+    cp ./result_plugin_c/lib/plugin_example.wasm lib &&
+    cp ./result_plugin_cpp/lib/plugin_example_cpp.wasm lib &&
+    chown `whoami` lib/* &&
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)